### DISCUSSION TITLE
Custom Module Navigation

### DISCUSSION THREAD

**Discussion by connorwinston** [directus/directus#9579] [Feature Requests]
Module Navigation used to be per Role, however, with the move towards having multiple roles set per user this was moved to be set for the whole project. I would like to propose that this be updated to have a filter that can be used. Creating multiple different module bar setups and each one has a filter that targets what users receive each one? That way it can still be customized for different users but can stay separate of roles.

**Comment by azrikahar** [comment]
> Module Navigation used to be per Role, however, with the move towards having multiple roles set per user this was moved to be set for the whole project.

In case anyone's wondering about this change, it was answered in #9076.

**Reply by Attacler** [reply]
> > Module Navigation used to be per Role, however, with the move towards having multiple roles set per user this was moved to be set for the whole project.
> 
> In case anyone's wondering about this change, it was answered in #9076.

I think what Connor proposes is a little differend than that.

**Reply by azrikahar** [reply]
> I think what Connor proposes is a little differend than that.

Yea no worries, that's why I didn't actually quote the whole thing or mark this as duplicate since Connor proposes new idea in the second half of his post 👍 It's just that `Module Navigation used to be per Role` seems to be a pretty frequent question recently so I tried to at least link to an answer for it.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
